README.md

```diff
@@ -71,36 +71,43 @@ Remember you'll need to run `bundle install` and `yarn install` on an ongoing ba
 
 ## Setup server
 
-### Manual Setup
+You can either run Solr/Postgres locally or spin them up in Docker containers
+with Lando.
+
+### Lando
+
+Lando will automatically set up docker images for Solr and Postgres which match
+the versions we use in Production. The ports will not collide with any other
+projects you're using Solr/Postgres for, and you can easily clean up with `lando
+destroy` or turn off all services with `lando poweroff`.
+
+1. Install Lando DMG from [[https://github.com/lando/lando/releases]]
+1. `lando start`
 
 1. For test:
    - `RAILS_ENV=test rake db:setup`
-   - `rake figgy:test`
    - In a separate terminal: `bundle exec rspec`
    - Run jest tests: `yarn test`
-2. For development:
+1. For development:
    - ``export SECRET_KEY_BASE=`rake secret` ``
    - `rake db:setup`
-   - In a separate terminal: `foreman start`
+   - In a separate terminal: `foreman start` (you can close the one launching solr)
```

Comment:
I'm not sure close the one launching solr is correct. Someone might have to comment it out. In general foreman doesn't start services in separate panes / windows / tabs.

Perhaps we should have a followup issue to update foreman to just run the web services, and documnt that the solr service will be separate in either case.

Reply:
Updated it. Foreman already isn't running solr for Figgy.